style/style.css

```diff
@@ -383,7 +383,7 @@ body > footer {
 		top: 0;
 		bottom: 0;
 		z-index: -1;
-		width: 50vw;
+		width: 49.6vw;
```

Comment:
`#quick-look pre::after` this acquires 50vw on both sides and it's container and `#quick-look pre.vanilla` & `#quick-look pre.bliss` both have border on their right and left respectively. I think that adds up to the width being more than 100vw? There is something else too but I couldn't find it.